lib/skate_web/controllers/swings_controller.ex

```diff
@@ -1,9 +1,6 @@
 defmodule SkateWeb.SwingsController do
   use SkateWeb, :controller
 
-  alias SkateWeb.AuthManager
-  alias Skate.Settings.RouteTab
-
   def index(conn, %{"route_ids" => route_ids}) do
```

Comment:
**question:** It doesn't really matter since it's an internal API, but I'm presuming if someone requests this route without passing `route_ids` they'll just get a 404 or some sort of error back?